anima/ui/user_dialog.py

```diff
@@ -0,0 +1,167 @@
+    """
+    def __init__(self,parent=maya_main_window()):
+        super(UserDialog,self).__init__(parent)
+        
+        self.setWindowTitle("User Dialog")
+        self.setWindowFlags(self.windowFlags()^QtCore.Qt.WindowContextHelpButtonHint)
+        
+        
+        self.create_widgets()
+        self.create_layouts()
+        self.create_connections()
+        
+    def create_widgets(self):
+        
+        self.dialog_label=QtWidgets.QLabel()
+        self.dialog_label.setStyleSheet("color: rgb(71, 143, 202);\n"
+                                   "font: 18pt;")        
+        self.dialog_label.setText("Create User")
+        
+        self.line = QtWidgets.QFrame()
+        self.line.setFrameShape(QtWidgets.QFrame.HLine)
+        self.line.setFrameShadow(QtWidgets.QFrame.Sunken)
+        
+        self.name_line_edit=QtWidgets.QLineEdit()
+        self.name_line_edit.setPlaceholderText("Enter Name")
+        
+        self.login_line_edit=QtWidgets.QLineEdit()
+        self.login_line_edit.setPlaceholderText("stalker")
+        
+        self.email_line_edit=QtWidgets.QLineEdit()
+        self.email_line_edit.setPlaceholderText("stalker@gmail.com")
+        
+        self.password_line_edit=QtWidgets.QLineEdit()
+        self.password_line_edit.setPlaceholderText("******")
+        
+        self.ok_button=QtWidgets.QPushButton("OK")
+        self.cancel_button=QtWidgets.QPushButton("Cancel")
+               
+       
+        
+    def create_layouts(self):
+
+        form_layout=QtWidgets.QFormLayout()
+        form_layout.addRow("Name", self.name_line_edit)
+        form_layout.addRow("Login", self.login_line_edit)
+        form_layout.addRow("Email", self.email_line_edit)
+        form_layout.addRow("Password", self.password_line_edit)
+        
+        button_layout=QtWidgets.QHBoxLayout()
+        button_layout.addStretch()
+        button_layout.addWidget(self.ok_button)
+        button_layout.addWidget(self.cancel_button)
+        
+        label_layout=QtWidgets.QVBoxLayout()
+        label_layout.addWidget(self.dialog_label)
+        label_layout.addWidget(self.line)
+        label_layout.addStretch()
+        
+        main_layout=QtWidgets.QVBoxLayout(self)
+        main_layout.addLayout(label_layout)
+        main_layout.addLayout(form_layout)
+        main_layout.addLayout(button_layout)
+        
+
+        self.resize(328,184)
+        
+        
+    def create_connections(self):
+        
+        self.ok_button.clicked.connect(self.check_userPass)
+        self.cancel_button.clicked.connect(self.close)
+        
+        
+    def check_userPass(self):
+        
+        self.user_name=self.name_line_edit.text()
+        self.user_login=self.login_line_edit.text()
+        self.user_email=self.email_line_edit.text()
+        self.user_password=self.password_line_edit.text()
+        
+        if len(self.user_name)<2 or len(self.user_login)<2 or len(self.user_email)<2 or len(self.user_password)<2 :
```

Comment:
Actually Stalker has validation controls for these kind of things. Check [auth.py](https://github.com/eoyilmaz/stalker/blob/master/stalker/models/auth.py) from Stalker.